src/resources/Subscriptions.ts

```diff
@@ -78,6 +78,7 @@ export type SubscriptionPlanItem =
 export type SubscriptionsListParams = CRUDPaginationParams & {
     search?: string;
     status?: SubscriptionStatus;
```

Comment:
Do we still keep the `status` ? Or are we gonna switch to using `statuses` only from now on?

Reply:
API did not remove it so we have no reason to not support it any longer if a customer wants to use it.

Reply:
yes, of course. Was just checking if the API removed it or not. 👍